siteimprove/admin/js/siteimprove-admin.js

```diff
@@ -89,6 +89,28 @@
 						$('#siteimprove_dev_mode_field').closest('tr').next('tr').hide();
 				}
 			);
+
+			$( '#siteimprove_disable_new_version_field' ).on(
```

Comment:
This here actually does it for disable_new_version. Do we want to hide stuff behind that?